core/src/utils/platform.ts

```diff
@@ -83,7 +83,7 @@ const isCapacitorNative = (win: any): boolean => {
 };
 
 const isElectron = (win: Window): boolean =>
-  testUserAgent(win, /electron/);
+  testUserAgent(win, /Electron/);
```

Comment:
could we make the regex case insensible instead?
```suggestion
  testUserAgent(win, /electron/i);
```

Reply:
Sure, good idea!